.circleci/config.yml

```diff
@@ -19,6 +19,9 @@ jobs:
           key: repo_{{ .Environment.CIRCLE_SHA1 }}
       - restore_cache:
           key: root_modules_circleci_node_10_{{ checksum "package.json" }}
+      - run:
+          name: Workaround for GoogleChrome/puppeteer#290
+          command: 'sh .circleci/setup_puppeteer.sh'
```

Comment:
This doesn't appear to have fixed the problem.

Reply:
Aye.

Reply:
I can't actually see it performing this step in the circleci logs though, which makes me suspicious.